### PR TITLE
Fix aspnet-session failing tests

### PR DESCRIPTION
**Pull request by petemounce**

### CODE REVIEW

File: NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs [hunk 1]
```diff
@@ -23,6 +24,7 @@ public AspNetSessionValueLayoutRendererTests()
 
         public void SetUp()
         {
+            ConfigurationItemFactory.Default.LayoutRenderers.RegisterDefinition("aspnet-session", typeof(AspNetSessionValueLayoutRenderer));
```
Comment (304NotModified) [inline NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs]: This should not be needed if nlog 4.0.1+ is used?

Comment (petemounce) [inline NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs]: I used both the resharper xunit test runner and the ncrunch test runner, and without that, I got 
```
System.ArgumentException: LayoutRenderer cannot be found: 'aspnet-session'
   at NLog.Config.Factory`2.CreateInstance(String name)
   at NLog.Layouts.LayoutParser.ParseLayoutRenderer(ConfigurationItemFactory configurationItemFactory, SimpleStringReader sr)
   at NLog.Layouts.LayoutParser.CompileLayout(ConfigurationItemFactory configurationItemFactory, SimpleStringReader sr, Boolean isNested, String& text)
   at NLog.Layouts.SimpleLayout.set_Text(String value)
   at NLog.Layouts.Layout.op_Implicit(String text)
   at NLog.Web.Tests.LayoutRenderers.AspNetSessionValueLayoutRendererTests.SessionWithCulture()
```

the packages.config shows NLog 4.0.1.

Comment (304NotModified) [inline NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs]: OK thanks, I will try to reproduce it locally first

Comment (304NotModified) [inline NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs]: tested it locally VS2013 update 5+R# 10.0.2: no problem;

Will try VS2015 also

Comment (304NotModified) [inline NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs]: @petemounce please add the following code to the erroneous test case

```
 InternalLogger.LogFile = @"C:\Temp\nlog.web.log";
 InternalLogger.LogLevel = LogLevel.Trace;
```

and post the log :)

Comment (petemounce) [inline NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs]: @304NotModified here you go - it's in petemounce/NLog.Web@2ba53a1921cd675f2885e4413c10a08d02cb2433. I branched from up-to-date master, made the code change, and have posted the log into that commit message.

Comment (304NotModified) [inline NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs]: @petemounce I added some comments there. Not sure if you have seen them. If not, I'm not in a hurry, so NP

**Comment by 304NotModified** [comment]
hey @petemounce, have you seen my comment https://github.com/petemounce/NLog.Web/commit/2ba53a1921cd675f2885e4413c10a08d02cb2433#commitcomment-15895368 ?

**Comment by 304NotModified** [comment]
Hi @petemounce , I think I have fixed the root cause with https://github.com/NLog/NLog.Web/pull/52

**Pull request closed without merging**